components/publish/StepHeader.tsx

```diff
@@ -6,7 +6,7 @@ export const StepHeader = styled<{ step?: number }>(
     return (
       <div {...rest}>
         {!!step && (
-          <StepChip scale={2} className="me-4">
+          <StepChip scale={2} className="me-4" style={{backgroundColor: "#1A3185", color: "white"}}>
```

Comment:
<details><summary>Here is the error logs from the vercel build. This is failing because StepChip props type doesn't include 'style'.</summary>
<p>

```
[12:41:53.237] Failed to compile.
[12:41:53.237] 
[12:41:53.237] ./components/publish/StepHeader.tsx:9:48
[12:41:53.237] Type error: No overload matches this call.
[12:41:53.237]   Overload 1 of 2, '(props: (({ ref?: Ref<Component<PropsWithChildren<{ active?: boolean | undefined; scale?: number | undefined; } & { className?: string | undefined; }>, any, any>> | undefined; ... 4 more ...; key?: Key | ... 1 more ... | undefined; } | { ...; }) & { ...; } & ({} | { ...; })) & { ...; }): ReactElement<...>', gave the following error.
[12:41:53.237]     Type '{ children: number; scale: number; className: string; style: { backgroundColor: string; color: string; }; }' is not assignable to type 'IntrinsicAttributes & ((({ ref?: Ref<Component<PropsWithChildren<{ active?: boolean | undefined; scale?: number | undefined; } & { className?: string | undefined; }>, any, any>> | undefined; ... 4 more ...; key?: Key | ... 1 more ... | undefined; } | { ...; }) & { ...; } & ({} | { ...; })) & { ...; })'.
[12:41:53.237]       Property 'style' does not exist on type 'IntrinsicAttributes & ((({ ref?: Ref<Component<PropsWithChildren<{ active?: boolean | undefined; scale?: number | undefined; } & { className?: string | undefined; }>, any, any>> | undefined; ... 4 more ...; key?: Key | ... 1 more ... | undefined; } | { ...; }) & { ...; } & ({} | { ...; })) & { ...; })'.
[12:41:53.238]   Overload 2 of 2, '(props: StyledComponentPropsWithAs<Comp<{ active?: boolean | undefined; scale?: number | undefined; }>, any, {}, never, Comp<{ active?: boolean | undefined; scale?: number | undefined; }>, Comp<{ active?: boolean | undefined; scale?: number | undefined; }>>): ReactElement<...>', gave the following error.
[12:41:53.238]     Type '{ children: number; scale: number; className: string; style: { backgroundColor: string; color: string; }; }' is not assignable to type 'IntrinsicAttributes & StyledComponentPropsWithAs<Comp<{ active?: boolean | undefined; scale?: number | undefined; }>, any, {}, never, Comp<{ active?: boolean | undefined; scale?: number | undefined; }>, Comp<...>>'.
[12:41:53.238]       Property 'style' does not exist on type 'IntrinsicAttributes & StyledComponentPropsWithAs<Comp<{ active?: boolean | undefined; scale?: number | undefined; }>, any, {}, never, Comp<{ active?: boolean | undefined; scale?: number | undefined; }>, Comp<...>>'.
[12:41:53.238] 
[12:41:53.238] [0m [90m  7 | [39m      [33m<[39m[33mdiv[39m {[33m...[39mrest}[33m>[39m[0m
[12:41:53.238] [0m [90m  8 | [39m        {[33m![39m[33m![39mstep [33m&&[39m ([0m
[12:41:53.238] [0m[31m[1m>[22m[39m[90m  9 | [39m          [33m<[39m[33mStepChip[39m scale[33m=[39m{[35m2[39m} className[33m=[39m[32m"me-4"[39m style[33m=[39m{{backgroundColor[33m:[39m [32m"#1A3185"[39m[33m,[39m color[33m:[39m [32m"white"[39m}}[33m>[39m[0m
[12:41:53.238] [0m [90m    | [39m                                               [31m[1m^[22m[39m[0m
[12:41:53.238] [0m [90m 10 | [39m            {step}[0m
[12:41:53.238] [0m [90m 11 | [39m          [33m<[39m[33m/[39m[33mStepChip[39m[33m>[39m[0m
[12:41:53.238] [0m [90m 12 | [39m        )}[0m
[12:41:53.355] error Command failed with exit code 1.
[12:41:53.355] info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
[12:41:53.380] Error: Command "yarn run vercel-build" exited with 1
```
</p>
</details> 

One way to resolve this error, since it's already a styled component, would be to extend it using styledcomponents when you need to: https://styled-components.com/docs/basics#extending-styles